crates/mongodb-connector/src/mongo_connector.rs

```diff
@@ -160,8 +161,6 @@ impl Connector for MongoConnector {
         let response = handle_query_request(state, v2_request)
             .await
             .map_err(mongo_agent_error_to_query_error)?;
-        let r = v2_to_v3_query_response(response);
-        tracing::warn!(v3_response = %serde_json::to_string(&r).unwrap());
```

Comment:
This fixes some debugging output I accidentally committed in a previous PR. I'd like to set up the sdk to making the logging level for this crate configurable so that I don't have to change `debug!`s to `warn!`s to see these traces.